pypeit/par/util.py

```diff
@@ -575,7 +575,8 @@ def make_pypeit_file(pypeit_file, spectrograph, data_files, cfg_lines=None, setu
     # Here we go
     with open(pypeit_file, 'w') as f:
         f.write('# Auto-generated PypeIt file\n')
-        f.write('# {0}\n'.format(time.strftime("%a %d %b %Y %H:%M:%S",time.localtime())))
+        #f.write('# {0}\n'.format(time.strftime("%a %d %b %Y %H:%M:%S",time.localtime())))
+        f.write('# {0}\n'.format(time.strftime("%Y-%m-%d",time.localtime())))
```

Comment:
I don't know if it is important, but it seems that now the local time is not printed.
Before:

```
# Auto-generated PypeIt file
# Fri 06 Nov 2020 16:18:47
```
Now:
```
# Auto-generated PypeIt file
# 2021-04-29
```

Reply:
Also `pypeit_obslog` generates a file with the data. That should be changed too.
```
# Auto-generated PypeIt Observing Log
# Thu 29 Apr 2021 15:17:57
```